kubernetes/linux/setup.sh

```diff
@@ -73,7 +73,8 @@ docker_cimprov_version=$(sudo tdnf list installed | grep docker-cimprov | awk '{
 echo "DOCKER_CIMPROV_VERSION=$docker_cimprov_version" >> packages_version.txt
 
 #install fluent-bit
-sudo tdnf install fluent-bit-3.1.9 -y
+sudo tdnf install azcu-fluent-bit-4.0.9 -y
+sudo tdnf install postgresql-libs openldap-libs openldap -y
```

Comment:
@wanlonghenry , why are these new dependencies for this version? are these coming from up stream?

Reply:
@wanlonghenry - did verify the functional and perf tests to ensure no functional and perf regression?